envs/doks/storage-class.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2019, Wazuh Inc.
+#
+# This program is a free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public
+# License (version 2) as published by the FSF - Free Software
+# Foundation.
+
+# Wazuh StorageClass
+
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: wazuh-storage
+  annotations:
+    cas.openebs.io/config:
+      - name: NFSServerType
```

Comment:
there is no need to use NFS inside the storage class since the implementation made on top of EKS is intended to use block storage

Reply:
Last time I used DOKS, they had an issue where you couldn't deploy storage with RWX (ReadWriteMany) permissions, so containers wouldn't start with a "Cannot attach volume" error. The proposed way from Digital Ocean is the following link is:

https://docs.digitalocean.com/products/marketplace/catalog/openebs-nfs-provisioner/

Maybe there is another way to fix this but I'm not really sure.